GraphicEditor/View/Windows/RoomInformation.xaml

```diff
@@ -8,26 +8,182 @@
         Title="RoomInformation" 
         Icon="/Resources/Icons/bed.png"
         WindowStartupLocation="CenterScreen"
-        d:DesignHeight="300" d:DesignWidth="500">
+        d:DesignHeight="500" d:DesignWidth="1000">
     
-    <Grid Background="AntiqueWhite">
+    <Grid Background="AliceBlue">
         
         <Grid.ColumnDefinitions>
             <ColumnDefinition Width="0.7*" />
             <ColumnDefinition Width="2*" />
         </Grid.ColumnDefinitions>
 
-        
-        <ComboBox ItemsSource="{Binding Path=Beds, Mode=TwoWay}"
+
+        <ListBox ItemsSource="{Binding Path=Beds, Mode=TwoWay}"
                   Name="roomComboBox"
-                  SelectedItem="{Binding Path=SelectedBed, UpdateSourceTrigger=PropertyChanged, Mode=TwoWay}"
+                  SelectedItem="{Binding Path=BedInfo, UpdateSourceTrigger=PropertyChanged, Mode=TwoWay}"
                   Grid.Column="0" 
                   Grid.RowSpan="3" 
                   Margin="20"
-                  FontSize="32"/>
+                  FontSize="14"
+                  Style="{StaticResource MaterialDesignToolVerticalToggleListBox}"
+                  Cursor="Hand">
+            <ListBox.ItemTemplate>
+                <DataTemplate>
+                    <TextBlock Text="{Binding Path=Name}"/>
+                </DataTemplate>
+            </ListBox.ItemTemplate>
+        </ListBox>
 
         <Grid Grid.Column="1" >
-            <Label  Content="{Binding Path=BedInfo, UpdateSourceTrigger=PropertyChanged, Mode=TwoWay}" FontSize="32" HorizontalAlignment="Center" VerticalAlignment="Center"> </Label>
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="auto"/>
+                <ColumnDefinition Width="*"/>
+            </Grid.ColumnDefinitions>
+            <Grid.RowDefinitions>
+                <RowDefinition Height="auto"/>
+                <RowDefinition Height="auto"/>
+                <RowDefinition Height="auto"/>
+                <RowDefinition Height="auto"/>
+                <RowDefinition Height="auto"/>
+                <RowDefinition Height="auto"/>
+                <RowDefinition Height="auto"/>
+                <RowDefinition Height="auto"/>
+            </Grid.RowDefinitions>
+
+            <!-- Top side -->
+
+            <TextBlock Grid.Column="0"
```

Comment:
You need to pick a convention here 😄 either comment - white space - element or comment - element